models/git/commit_status.go

```diff
@@ -36,7 +36,7 @@ type CommitStatus struct {
 	SHA         string                 `xorm:"VARCHAR(64) NOT NULL INDEX UNIQUE(repo_sha_index)"`
 	TargetURL   string                 `xorm:"TEXT"`
 	Description string                 `xorm:"TEXT"`
-	ContextHash string                 `xorm:"char(40) index"`
+	ContextHash string                 `xorm:"char(64) index"`
```

Comment:
We need a migration since the struct changed.

Reply:
Migration is now added and I tested it with postgres.